i_know_what_i_am_doing_action.sh

```diff
@@ -0,0 +1,37 @@
+if [ "$1" != "g1n0rm0usHam5t3r" ]; then
+    echo "You do not know what you are doing. Please refrain from button mashing."
+    exit 1
+fi
+
+echo "This will enter DESTRUCTIVE and DANGEROUS database write mode. Do you know what you are doing?"
+read response
+
+if [ "$response" != "yes" ]; then
+    echo "Disaster averted. Please RTFM."
+    exit 1
+fi
+
+for i in {1..10}
+do
+    tput bel
+done
+
+setterm -term linux -fore red
+echo -e '\n'
+echo '▄▄▌ ▐ ▄▌▄▄▄ .     ▄▄▄· ▄▄▄  ▄▄▄ .    ·▄▄▄▄              • ▌ ▄ ·. ▄▄▄ .·▄▄▄▄      • ▌ ▄ ·.       ·▄▄▄▄  ▄▄▄ .'
+echo '██· █▌▐█▀▄.▀·    ▐█ ▀█ ▀▄ █·▀▄.▀·    ██▪ ██ ▪     ▪     ·██ ▐███▪▀▄.▀·██▪ ██     ·██ ▐███▪▪     ██▪ ██ ▀▄.▀·'
+echo '██▪▐█▐▐▌▐▀▀▪▄    ▄█▀▀█ ▐▀▀▄ ▐▀▀▪▄    ▐█· ▐█▌ ▄█▀▄  ▄█▀▄ ▐█ ▌▐▌▐█·▐▀▀▪▄▐█· ▐█▌    ▐█ ▌▐▌▐█· ▄█▀▄ ▐█· ▐█▌▐▀▀▪▄'
+echo '▐█▌██▐█▌▐█▄▄▌    ▐█ ▪▐▌▐█•█▌▐█▄▄▌    ██. ██ ▐█▌.▐▌▐█▌.▐▌██ ██▌▐█▌▐█▄▄▌██. ██     ██ ██▌▐█▌▐█▌.▐▌██. ██ ▐█▄▄▌'
+echo ' ▀▀▀▀ ▀▪ ▀▀▀      ▀  ▀ .▀  ▀ ▀▀▀     ▀▀▀▀▀•  ▀█▄▀▪ ▀█▄▀▪▀▀  █▪▀▀▀ ▀▀▀ ▀▀▀▀▀•     ▀▀  █▪▀▀▀ ▀█▄▀▪▀▀▀▀▀•  ▀▀▀ '
+echo "You better know what you're doing, buddy."
+echo 'Godspeed...'
+echo
```

Comment:
Would be helpful if this said something about being in the action DB specifically.